rawkit/__init__.py

```diff
@@ -2,3 +2,10 @@
 """
 The current version of the `rawkit` package.
 """
+
+from libraw.bindings import LibRaw
+
+_libraw = LibRaw()
```

Comment:
We should just ditch the global reference and construct LibRaw objects as we need them. This will also help with testing as we can mock the library *before* LibRaw is instantiated.